src/Template/Element/sql_log_panel.ctp

```diff
@@ -84,3 +114,57 @@ SqlFormatter::$pre_attributes = 'style="color: #222; background-color: transpare
 <?php if ($noOutput): ?>
 <div class="warning"><?= __d('debug_kit', 'No active database connections') ?></div>
 <?php endif ?>
+
+<script>
+$(document).ready(function() {
+    var baseUrl = '<?= $this->Url->build([
+        'plugin' => 'DebugKit',
+        'controller' => 'Toolbar',
+        'action' => 'sqlExplain'
+    ]); ?>';
+
+    function createTable(result) {
+        var $table = $('<table>');
```

Comment:
Ah sorry. It's my personal habit 😛 When I assign an instance of jQuery object assign to a variable, I always prefix `$` to the name. I will remove all of them.